Gifski/ConversionCompletedViewController.swift

```diff
@@ -70,6 +70,10 @@ final class ConversionCompletedViewController: NSViewController {
 				return
 			}
 
+			guard self.view.window?.contentViewController == self else {
+				return
+			}
+
 			self.tooltip.show(from: self.draggableFile, preferredEdge: .maxY)
```

Comment:
This is the fix for the tooltip crash. It's an odd one and quite difficult to reproduce.